SwiftMonkeyPaws/MonkeyPaws.swift

```diff
@@ -35,10 +31,60 @@ private let circleRadius: CGFloat = 7
     }
     ```
 */
+
 public class MonkeyPaws: NSObject, CALayerDelegate {
+
+    public struct Configuration {
+        // Customise the appearance of the paws
+        public struct Paws {
```

Comment:
The default `struct` initializer is `internal` therefore it's not possible to instantiate this object without an explicit `public init()` defined.